cmd/misc/main.go

```diff
@@ -359,6 +362,10 @@ func main() {
 		}
 	case "export-stats-totals":
 		exportStatsTotals(opts.Columns, opts.StartDay, opts.EndDay, opts.DataConcurrency)
+	case "export-sync-committee-periods":
+		exportSyncCommitteePeriods(rpcClient, bt, opts.StartDay, opts.EndDay, opts.DryRun)
+	case "export-sync-committee-validator-stats":
+		exportSyncCommitteeValidatorStats(rpcClient, bt, opts.StartDay, opts.EndDay, opts.DryRun, true)
```

Comment:
I've talked with StefanF but he knows the misc tool more from a usage perspective rather than a technical one. Could you point me in the direction where this routine is you are talking about? I see that there is a `sync_duties_exported` but I found no usage for it outside of writing it after the export is done.

Reply:
Nvm, reexport tools are gone and the existence of `exportSyncCommitteeValidatorStats` is valid.